core/class-fire-resources_styles.php

```diff
@@ -686,7 +686,7 @@ function czr_fn_maybe_write_boxed_layout_inline_css( $_css ) {
                //Add some rules from sm up
                //add padding
                $css_container[]        = sprintf( '@media (min-width: %1$spx){ %2$s{ padding-right: %3$spx; padding-left:  %3$spx; } }',
-                           $css_container_widths[ 'sm' ],
+                           $css_mq_breakpoints[ 'sm' ],
```

Comment:
Was that change related to a bug reported for something else ?
```suggestion
                           $css_mq_breakpoints[ 'sm' ],
```

Reply:
something I noticed, that was wrong

Reply:
the previous code generated a media query breakpoint too small.

Reply:
ok ! thx